sample/SampleHost/Program.cs

```diff
@@ -10,6 +11,8 @@ class Program
         static void Main(string[] args)
         {
             var config = new JobHostConfiguration();
+            config.Queues.VisibilityTimeout = TimeSpan.FromSeconds(15);
```

Comment:
For my local testing. With these changes, when the sample function throws an exception, there is a 15 second delay before the next retry now.